solr/bin/solr

```diff
@@ -2004,7 +2012,7 @@ if [ "${SOLR_LOG_PRESTART_ROTATION:=false}" == "true" ]; then
   # Enable any of these if you require old remove/archive behavior
   #run_tool utils -s "$DEFAULT_SERVER_DIR" -l "$SOLR_LOGS_DIR" $q -remove_old_solr_logs 7 || echo "Failed removing old solr logs"
```

Comment:
Did you intend to edit the line above?

Reply:
Oops, yes i meant to edit the `archive_gc_logs` instead of the `archive_console_logs` line, good catch!

Reply:
Nit-pick, the `||` used to be aligned above eachother :)